### PR TITLE
Login + Signup + NotFound High fidelity Implementations

### PR DESCRIPTION
**Pull request by parth4apple**
## Changes
- Login + Signup + NotFound now match high fidelity
- Purged old way to call authcontext -- using only useAuthContext() now
- Create Account doesnt auto log user in now
- Limit modal height so it doesnt look odd on larger screens

## TODO
- user real avatars instead of just one placeholder one -- waiting on design for finalized icons
- forgot password feature -- should email user with instructions to reset password

### CODE REVIEW

**Review by paulkid298 — APPROVED** [review]

File: src/Profile/ProfilePage.js [hunk 1]
```diff
@@ -15,32 +18,55 @@ var review_list = [
...
+  const { user_id } = useParams();
+  let navigate = useNavigate();
+
 
   // TODO: add profile call here -- based on user_id
-  componentDidMount = () => {
-    console.log(this.props.params.user_id)
-  };
+  useEffect(() => {
+    console.log(user_id);
```
Comment (wes-chen) [inline src/Profile/ProfilePage.js:35]: can you add a TODO to remove this later, along with the other console logs, so we don't have anything emitting in prod

File: src/User/Signup.js [hunk 2]
```diff
@@ -1,62 +1,122 @@
...
+          </BoldText>
+          <Input
+            type="text"
+            name="uname"
+            placeholder="Username"
+            value={uname}
+            onChange={(e) => setUname(e.target.value)}
+            required
+          />
+          <Input
```
Comment (wes-chen) [inline src/User/Signup.js:84]: is it possible to do an email check here? like has to have an @ and a . probably

**Review by wes-chen — CHANGES_REQUESTED** [review]
looks good on first glance, but missing one feature

File: src/Profile/ProfilePage.js [hunk 3]
```diff
@@ -15,32 +18,55 @@ var review_list = [
...
 
-class ProfilePage extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      reviews: review_list
-    };
-  }
+function ProfilePage() {
+  const [reviews, setReviews] = useState(review_list);
```
Comment (wes-chen) [inline src/Profile/ProfilePage.js:27]: so we don't have the part where we actually pull the reviews from the profile page right

File: src/Profile/ProfilePage.js [hunk 4]
```diff
@@ -15,32 +18,55 @@ var review_list = [
   "restroom_id4",
   "restroom_id5",
   "restroom_id6",
-  "restroom_id7", 
+  "restroom_id7",
   "restroom_id8",
-  "restroom_id9"
+  "restroom_id9",
```
[the remark this answers is not in the document]
Comment (parth4apple) [reply src/Profile/ProfilePage.js:23]: i think my prettier just does that

File: src/Profile/ProfilePage.js (hunk 3 above)
Comment (parth4apple) [reply src/Profile/ProfilePage.js:27]: i think we do at /profile/comments/:user_id

Comment (wes-chen) [reply src/Profile/ProfilePage.js:26]: then can you add the axios call here then and update the state

**Review by wes-chen — APPROVED** [review]
one call left i think because it triggers a warning still that setReviews is declared but never used

**Pull request merged**